### DISCUSSION TITLE
疑难解答专用讨论

### DISCUSSION THREAD

**Discussion by langningchen** [XMOJ-Script-dev/XMOJ-Script#39] [General]
可以在此问我问题

**Comment by langningchen** [comment]
每个问题一个 Comment ，如果要补充内容或者我要回答都会用 Reply 的形式，注意要 @ 我以便让我收到通知

**Comment by PythonSmall-Q** [comment]
那个仓库怎么从PythonSmall-Q安装redir到xmoj-script-dev有点烦 @langningchen

**Reply by langningchen** [reply]
直接用 GitHub 的 Transfer

**Reply by langningchen** [reply]
@PythonSmall-Q

**Reply by langningchen** [reply]
我记得仓库设置里面最下面有一个更换拥有者的

**Reply by langningchen** [reply]
这样 GitHub 能自动帮你 302

**Reply by PythonSmall-Q** [reply]
不行，不能转换到团队

**Reply by langningchen** [reply]
那就不行了 @PythonSmall-Q

**Reply by langningchen** [reply]
你可以先改用户脚本里的链接，慢慢再转移过去

**Reply by boomzero** [reply]
https://github.com/XMOJ-Script-dev/XMOJ-Script/issues/33#issuecomment-1744621716

**Reply by PythonSmall-Q** [reply]
转了成功

**Comment by boomzero** [comment]


**Reply by langningchen** [reply]
> 每个问题一个 Comment ，如果要补充内容或者我要回答都会用 Reply 的形式，注意要 @ 我以便让我收到通知

请符合规范！ @boomzero

**Comment by boomzero** [comment]
GitHub 有bug, 就是那个quote reply

**Reply by langningchen** [reply]
有什么 BUG ？

**Reply by langningchen** [reply]
哦，不过你可以不用他的 Quote reply

**Reply by langningchen** [reply]
还有你这个用邮件回复的也不符合规范

**Reply by langningchen** [reply]
不过就这样吧
